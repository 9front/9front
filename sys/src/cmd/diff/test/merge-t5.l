1
a
b
c
2
3
4
x
6
7
8
9
10
