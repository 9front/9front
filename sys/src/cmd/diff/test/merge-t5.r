1
2
3
w
x
y
z
8
9
10
