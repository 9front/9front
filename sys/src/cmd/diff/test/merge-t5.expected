1
a
b
c
2
3
<<<<<<<<<< merge-t5.l
4
x
6
7
========== original
4
5
6
7
========== merge-t5.r
w
x
y
z
>>>>>>>>>>
8
9
10
